extensions/systemd/tests/ConsumeJournaldTest.cpp

```diff
@@ -176,7 +176,7 @@ TEST_CASE("ConsumeJournald", "[consumejournald]") {
     libwrapper_observer->journal.emplace_back("systemd", "Mounted /boot.", 1);
     plan->runCurrentProcessor();
     REQUIRE("6" == get_cursor_position());
-    REQUIRE("2021-04-15T17:17:09.123456000+00:00 test-pc systemd[1]: Mounted /boot." == plan->getContent(plan->getCurrentFlowFile()));
+    REQUIRE("2021-04-15T17:17:09.123456+00:00 test-pc systemd[1]: Mounted /boot." == plan->getContent(plan->getCurrentFlowFile()));
```

Comment:
On Debian with clang-11, this is the date format that came out of the old code, making the string comparison and the test fail. I decided to restrict the precision to microseconds for all platforms. That's why ConsumeJournald is changed.